### ISSUE TITLE
Put all important things in the initial post?

### ISSUE THREAD

**Issue by jenstornell** [k-next/plugins#69]
There are a lot of discussion and it takes time to follow them all.

## Suggestion

Every time we all agree on something, for example how a plugin should be built. Then place that info in the initial post/issue. Like a summery. Maybe include important pitfalls.

Also if someone started a plugin, put a link to that in the initial post/issue as well. Maybe two developers are building two plugins. Then two links will be in the initial post/issue, maybe in a list.

**Comment by bnomei** [comment]
yes. i agree with jens. we should split actual plugin and discussion about how to solve a problem it better.
